mthamayor/politico
# [Finishes #163500866] Create political party API

**What this PR does**
Creation of API which enables admins to create new political parties
**Description**
Enables admins to create new political parties, by inserting the following request body in the provided form: name, party headquarters, and logo URL.

**Testing procedures**
1. start up the local server
2. copy the API URL: http://localhost:5000/api/v1/parties and paste it in a Postman POST request.
3. Fill up the form body to contain the following
![screenshot 74](https://user-images.githubusercontent.com/16121915/51919346-83f60e80-23e3-11e9-94ea-7578913bd616.png)
4. it should return a success response

**Pivotal tracker story**
https://www.pivotaltracker.com/story/show/163500866
